src/django_upgrade/fixers/model_field_choices.py

```diff
@@ -0,0 +1,58 @@
+"""
+Drop `.choices` for model field `choices` parameters:
+https://docs.djangoproject.com/en/5.0/releases/5.0/#forms
+"""
+from __future__ import annotations
+
+import ast
+from functools import partial
+from typing import Iterable
+
+from tokenize_rt import Offset
+from tokenize_rt import Token
+
+from django_upgrade.ast import ast_start_offset
+from django_upgrade.data import Fixer
+from django_upgrade.data import State
+from django_upgrade.data import TokenFunc
+from django_upgrade.tokens import find_last_token
+
+fixer = Fixer(
+    __name__,
+    min_version=(5, 0),
+)
+
+
+@fixer.register(ast.Call)
+def visit_Call(
+    state: State,
+    node: ast.Call,
+    parents: list[ast.AST],
+) -> Iterable[tuple[Offset, TokenFunc]]:
+    if (
+        not state.looks_like_migrations_file
```

Comment:
Let's instead introduce `looks_like_models_file` which checks for the name `models.py` or a directory `models` in the path. That'll be more precise and avoid running this on many unnecessary function calls.

Reply:
Should I do the same in `null_boolean_field.py` ?

Reply:
I went ahead and did that, let me know if you think I should revert that